assets/src/scripts/_file-click.js

```diff
@@ -80,9 +81,42 @@ const fileClick = async ({ fileName, metadata, url }) => {
             </ul>`
           : ""
       }
+      <li>
+        <ul class="mt-2">
+          <li>
+            <strong>Review:</strong>
```

Comment:
I don't think "Review" is needed here. The button is well labelled, so it's clear enough.

Reply:
There will be a comments box and maybe a reject button too in this section, I think?

I'm just following the DOM structure Tom had created for the previous two segments.  I think probably best for him to adjust when he's back as part of making in look nice?